integration_tests/plugins/test_rust_plugin.py

```diff
@@ -101,6 +101,18 @@ def test_cross_compiling(self):
         binary = os.path.join(self.parts_dir, 'rust-hello', 'install', 'bin',
                               'rust-hello')
         self.assertThat(binary, HasArchitecture('aarch64'))
+        self.assertThat(binary, HasLinkage('dynamically linked'))
+
+    def test_cross_compiling_musl(self):
```

Comment:
Why no integration test for normal compillation?

Reply:
I figured the existing ones are enough... I added the linkage assertion to the existing first test now.

Reply:
Done